src/components/common/AdminMenu.tsx

```diff
@@ -34,8 +35,12 @@ const StyledMenu = styled(Menu)`
     background: none !important;
   }
 `
+export type renderMemberAdminMenuProps = {
+  menuProps: MenuProps
+  defaultMenuItems: { key: string; item: React.ReactElement | boolean | undefined }[]
+}
```

Comment:
it may only need to use the item's key, icon, name... as props, not a ReactElement

Reply:
the `defaultMenuItems` is to expose our MenuItem to the developer, so we need to send the ReactComponent